06-conclusion.Rmd

```diff
@@ -96,20 +100,17 @@ Data visualization allows decision makers to make sense out of the visible patte
 
 * **Deliver value in your visualization.** Many graphs may be aesthetically pleasing or exciting but deliver no value. A graph with good value should deliver important insights to aid decision-making or prompt an action. Keep in mind, the main objective of a visualization is to deliver value, and each design decision should help achieve this.
  
-* **Understanding your audience is the most difficult and important part of developing a visualization.** Different audiences will feel differently about the same chart. So when we develop the argument from the data, we should keep our intended audience in mind.
+* **Understanding your audience is the most difficult and important part of developing a visualization.* Different audiences will feel differently about the same chart. So when we develop the argument from the data, we should keep our intended audience in mind.
 
-* **Share what you have found.** A visualization can be an effective influencer if used in the correct way and at the correct time. 
+* **Share what you have found** A visualization can be an effective influencer if used in the correct way and at the correct time. 
```

Comment:
Also, all the points in the bold statements end with a full stop. 
Should we keep it consistent and add (.) after bold statements?

Reply:
> Also, all the points in the bold statements end with a full stop.
> Should we keep it consistent and add (.) after bold statements?

I agree with this. I looked thru the other points and they all end with a period. I think we should keep the period here as well.

Reply:
hey guys. Thanks for pointing this out. Will get this resolved.